Makefile

```diff
@@ -26,13 +26,19 @@ $(build)/$(function)/message.c:
 	mkdir -p $(build)/$(function)
 	cp $(function)/src/main/proto/* $(build)/$(function)
 
-$(build)/$(function)/$(wasm): $(build)/$(function)/message.c $(build)/lib/libprotoc.a 
+$(build)/$(function)/$(wasm): $(build)/$(function)/message.c $(build)/lib/libprotoc.a
 	mkdir -p $(build)/$(function)
-	cd $(build)/$(function) && emcc -I ../include -s ERROR_ON_UNDEFINED_SYMBOLS=0 -Os -s STANDALONE_WASM -s EXPORTED_FUNCTIONS="['_filter','_malloc','_free']" -Wl,--no-entry message.c message.pb-c.c ../lib/libprotobuf-c.a ../lib/libprotobuf.a -o message.wasm
+	cd $(build)/$(function) && emcc -I ../include -s ERROR_ON_UNDEFINED_SYMBOLS=0 -Os -s STANDALONE_WASM -s EXPORTED_FUNCTIONS="['_filter','_malloc','_free']" -Wl,--no-entry message.c cloudevents.pb-c.c any.pb-c.c timestamp.pb-c.c ../lib/libprotobuf-c.a ../lib/libprotobuf.a -o message.wasm
 
 $(function): $(build)/$(function)/$(wasm)
 	cp $(build)/$(function)/$(wasm) $(function)/src/main/resources
 
+$(function)/protoc:
```

Comment:
This is a manual target (not included in `ALL` or `function`) to easily compile+generate the `CloudEvent` proto C files. I mentioned it in the main README as well.